.gitignore

```diff
@@ -1,2 +1,23 @@
 .DS_Store
 /target
+
+# Generated by Cargo
+# will have compiled files and executables
+debug/
+target/
```

Comment:
These two seem redundant with `/target` above. Can you remove them and put the comment above `/target`? Actually, put `/target/`,

Reply:
Done.